packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -64,14 +66,41 @@ export function Comments( {
 		setIsConfirmDialogOpen( false );
 	};
 
-	const blockCommentId = useSelect( ( select ) => {
+	const blockCommentId = useSelect( () => {
```

Comment:
I think you removed the select on accident here

Reply:
`error  'select' is already declared in the upper scope on line 20 column 34  no-shadow`

@ellatrix  - I removed the select to fix the above linting issue.

Reply:
You should remove the `select` import above, and then you can pass select as an argument.

Example: `useSelect( ( select ) => {}, [] );`